tests/Expected/Business/testJsonSerialize.json

```diff
@@ -0,0 +1,42 @@
+{
+  "days": {
+    "1": {
+      "dayOfWeek": 1,
+      "openingIntervals": [
+        {
+          "start": {
+            "hours": 9,
+            "minutes": 0,
+            "seconds": 0
+          },
+          "end": {
+            "hours": 13,
+            "minutes": 0,
+            "seconds": 0
+          }
+        },
+        {
+          "start": {
+            "hours": 14,
+            "minutes": 0,
+            "seconds": 0
+          },
+          "end": {
+            "hours": 17,
+            "minutes": 0,
+            "seconds": 0
+          }
+        }
+      ]
+    },
+    "5": {
+      "dayOfWeek": 5,
+      "openingIntervals": []
```

Comment:
Sorry for the delay @dragosprotung, I've been busy with other projects.

I think calling `evaluateOpeningIntervals` with the current date should be ok, so we don't have to make it public.